hello_world_ironsketch.cc

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
```

Comment:
Please change your script file to be in line with #1292

Reply:
I'm sorry I am trying to learn all this, I updated it but when I try to push it, it fails. It says the remote has something I don't. So I tried to pull to update everything. And it still doesn't work :/